dev-libs/glib/glib2_bootstrap-2.81.0.recipe

```diff
@@ -0,0 +1,238 @@
+platform, thus creating GLib as a separate product. GLib was released as a \
+separate library so other developers, those who did not make use of the \
+GUI-related portions of GTK+, could make use of the non-GUI portions of the \
+library without the overhead of depending on the entire GUI library.
+Since GLib is a cross-platform library, applications using it to interface \
+with the operating system are usually portable across different operating \
+systems without major changes."
+HOMEPAGE="https://www.gtk.org/"
+COPYRIGHT="1995-1997  Peter Mattis, Spencer Kimball and Josh MacDonald
+	1991-2003 Free Software Foundation, Inc.
+	1997-2006 University of Cambridge.
+	1998-2001, 2003-2010 Red Hat, Inc.
+	2007-2009 Nokia Corporation
+	2008, 2010 Oracle and/or its affiliates, Inc. All rights
+	2008-2010 Codethink Limited
+	2008-2010 Collabora Ltd.
+	1995-2010 Several others"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://gitlab.gnome.org/GNOME/glib/-/archive/$portVersion/glib-$portVersion.tar.gz"
+CHECKSUM_SHA256="e9c311a25abb5717fb83f0f3a36daddc039e61fe37cea61cad4d0e455a6cdca0"
+SOURCE_DIR="glib-$portVersion"
+srvGitRev2="0854af0fdb6d527a8d1999835ac2c5059976c210"
+SOURCE_URI_2="https://gitlab.gnome.org/GNOME/gvdb/-/archive/0854af0fdb6d527a8d1999835ac2c5059976c210/gvdb-$srvGitRev2.tar.gz"
+CHECKSUM_SHA256_2="08352e54e8216d9001820c627c62858585465b51dc557cc22f0f4770ed182ebd"
+SOURCE_DIR_2="gvdb-$srvGitRev2"
+giVer="1.81.2"
+SOURCE_URI_3="http://ftp.gnome.org/pub/GNOME/sources/gobject-introspection/${giVer%.*}/gobject-introspection-$giVer.tar.xz"
+CHECKSUM_SHA256_3="ec973bf5c2c2e69d14642df38e508c24ae91a254b6c38d2828022c71f2a144e0"
+SOURCE_DIR_3="gobject-introspection-$giVer"
+PATCHES_3="gobject_introspection-$giVer.patchset"
+
+PATCHES="glib2-$portVersion.patchset"
+
+ARCHITECTURES="?all !x86_gcc2"
+SECONDARY_ARCHITECTURES="?x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+libVersion="0.8100.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	glib2_bootstrap$secondaryArchSuffix = $portVersion
+	cmd:gapplication$commandSuffix
+	cmd:gdbus$commandSuffix
+	cmd:gdbus_codegen$commandSuffix
+	cmd:gi_compile_repository$commandSuffix
+	cmd:gi_decompile_typelib$commandSuffix
+	cmd:gi_inspect_typelib$commandSuffix
+	cmd:gio$commandSuffix
+	cmd:gio_launch_desktop$commandSuffix
+	cmd:gio_querymodules$commandSuffix
+	cmd:glib_compile_resources$commandSuffix
+	cmd:glib_compile_schemas$commandSuffix
+	cmd:gresource$commandSuffix
+	cmd:gsettings$commandSuffix
+	lib:libgio_2.0$secondaryArchSuffix = $libVersionCompat
+	lib:libgirepository_2.0$secondaryArchSuffix = $libVersionCompat
+	lib:libglib_2.0$secondaryArchSuffix = $libVersionCompat
+	lib:libgmodule_2.0$secondaryArchSuffix = $libVersionCompat
+	lib:libgobject_2.0$secondaryArchSuffix = $libVersionCompat
+	lib:libgthread_2.0$secondaryArchSuffix = $libVersionCompat
```

Comment:
Is this similar to freetype_bootstrap? In this case, I think this shouldn't actually provide anything besides `glib2_bootstrap$secondaryArchSuffix`, so that it isn't picked up by other recipes accidentally instead of the real one, and it doesn't need a separate devel package.